spec/kitchen/driver/gce_spec.rb

```diff
@@ -92,7 +92,8 @@
         zone_name: nil,
         google_key_location: nil,
         google_json_key_location: nil,
-        preemptible: false
+        preemptible: false,
+        auto_restart: false
```

Comment:
Ah, sorry, misread - disregard the previous comment.

If auto_restart really should be set to true here, then we'll really have to add some proper validation logic, because preemptible will fail when set (albeit GCE will respond with a human-readable message which option needs to be set). However, I don't really see why, since the machines here are supposed to be short-lived, no? I fully accept that I may be missing something in the picture, though.

My reasoning to false - it was always set like that anyway, since Fog has the option set on all created machines by default, I only asked the author of the PR to add it so we have the parameter here in case somebody wants to set it to true and for consistency sake (since preemptible depends on the option).